torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -369,7 +369,7 @@ def parallelize_llama(model, world_mesh, parallel_dims, job_config: JobConfig):
     # apply AC + torch.compile
     ac_config = job_config.activation_checkpoint
     enable_compile = job_config.training.compile
-    for layer_id, transformer_block in model.layers.items():
+    for layer_id, transformer_block in model.layers.named_children():
```

Comment:
is there any ordering gaurantee of named_children though?
i think this will work only because the names are assumed to be '0', '1', ...

Reply:
actually, does order even matter here? i was thinking more about the model's forward code, but that code doesn't get affected by this.